cmake/org.eclipse.cdt.cmake.core/src/org/eclipse/cdt/cmake/core/internal/CMakePropertiesController.java

```diff
@@ -41,6 +41,7 @@
  * us to delete file CMakeCache.txt to avoid complaints by cmake.
  * @author Martin Weber
  */
+@Deprecated
```

Comment:
Lets delete instead of deprecating internal code.

Reply:
CMakePropertiesEvolutionTest is to test how it evolves when serializing, it does not do anything useful without serializing AFAICT